src/utils/ui-utils.ts

```diff
@@ -94,6 +98,12 @@ export function attachContainerHeightSensors(parentContainer?: any, parentContai
           resetHeight();
         }
       }
+      if (onContainerHeightChange) {
+        //To notify height change
+        onContainerHeightChange({
+          top: containerSticky.offsetHeight + parentContainerSticky.offsetHeight,
```

Comment:
Instead of recalculating this, is it possible to change the code above to first calculate the top. And then sets the container height using this?

What you have is not working properly for popups. You can test this on `isa:dataset` table (https://dev.derivacloud.org/chaise/recordset/#1/isa:dataset@sort(accession::desc::). For example the `Title` facet is a good example.